vaadin-grid-pro-flow-integration-tests/src/test/java/com/vaadin/flow/component/gridpro/test/BasicIT.java

```diff
@@ -26,6 +26,12 @@ public void editColumnsAdded() {
         Assert.assertEquals(columns.size(), 3);
     }
 
+    @Test
+    public void customRepresentationIsRendered() {
+        GridTHTDElement cell = grid.getCell(0, 2);
+        Assert.assertEquals("No", cell.$("span").first().getText());
+    }
+
```

Comment:
Should be tested with other types of renderers also

Reply:
Actually it's not needed. If one renderer works, we can expect the others to work too, as long as `Grid` tests verify those.